### ISSUE TITLE
New issue label suggestion- Needs User Feedback

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#2566]
Sometimes an issue is opened by a user, or else for something a user has reported, and then we never hear back from them with steps to reproduce or whatever other info we need.

Realizing that this has a bit of overlap with "Needs Reproducing," we need a way to quickly go through old issues that should be closed because we're unable to obtain info from the user necessary to the advancement of the issue.

**Comment by arnaudbroes** [comment]
I agree and I think @wpsmort agreed on this topic previously at some point when it was brought up. 

I've created "Needs User Input" in both free and Pro (for example, when waiting on input from someone in the premium forums).
